activiti-cloud-service-common/activiti-cloud-services-dbp-rest/src/main/java/org/activiti/cloud/alfresco/config/AlfrescoWebAutoConfiguration.java

```diff
@@ -50,13 +50,19 @@ public class AlfrescoWebAutoConfiguration implements WebMvcConfigurer {
 
     private final PageableHandlerMethodArgumentResolver pageableHandlerMethodArgumentResolver;
     private final int defaultPageSize;
+    private final int maxItemsLimit;
+    private final boolean maxItemsLimitEnabled;
 
     public AlfrescoWebAutoConfiguration(
         @Lazy PageableHandlerMethodArgumentResolver pageableHandlerMethodArgumentResolver,
-        @Value("${spring.data.rest.default-page-size:100}") int defaultPageSize
+        @Value("${spring.data.rest.default-page-size:100}") int defaultPageSize,
```

Comment:
why default it's 100 if usually in UI we show 25 results?

Reply:
As we agreed i'll change the default to 25 results